exercises/01.styling/01.solution.public-links/tests/e2e/favicon.test.ts

```diff
@@ -3,6 +3,6 @@ import { test, expect } from '@playwright/test'
 test('checks favicon', async ({ page }) => {
 	await page.goto('/')
 	const favicon = await page.$('link[rel="icon"]')
-	const href = await favicon?.getAttribute('href')
+	const href = await favicon?.getAttribute('href') ?? null
```

Comment:
Rather than a conditional in our test, let's use a better assertion that covers the undefined case

Reply:
I updated it to use a `toBeTruthy` assertion. In this case it covers 3 possibilities: an empty string, null and undefined.

IMO, it is vaguely express the test, "A link href to be truthy?".